src/components/Footer.js

```diff
@@ -139,171 +140,249 @@ const ComBox = styled.div`
+                                    <ul className="list">
+                                        <List><a href="Howitworks">How it works</a></List>
+                                        <List><a href="Measurement&Installation">Measurement & Installation</a></List>
+                                        <List><a href="Papertypes">Paper types</a></List>
+                                        <List><a href="Delivery&returns">Delivery & returns</a></List>
+                                        <List><a href="faq">Frequently asked questions</a></List>
+                                        <List><a href="Ordersamples">Order samples</a></List>
+                                    </ul>
+                                </StyledColumn3>
+                            </StyledLink>
                         {/* Column4 */}
                         <StyledColumn4>
-                            <div className="shop-worldwide">
-                                <h3 className="footer-titles">Shop worldwide</h3>
-                                <div className="flag-box">
-                                    <img src={flag0} alt="flag0"/>
-                                    <img src={flag1} alt="flag1"/>
-                                    <img src={flag2} alt="flag2"/>
-                                    <img src={flag3} alt="flag3"/>
-                                    <img src={flag4} alt="flag4"/>
-                                    <img src={flag5} alt="flag5"/>
-                                    <img src={flag6} alt="flag6"/>
-                                    <img src={flag7} alt="flag7"/>
-                                    <img src={flag8} alt="flag8"/>
-                                    <img src={flag9} alt="flag9"/>
-                                </div>
-                            </div>
-                            <div className="secure-shopping">
-                                <h3 className="footer-titles">Secure shopping</h3>
-                                    <div className="norton-space">
-                                        <img className="norton" src={secure1} alt="secure1"/>
-                                    </div>
-                                <div className="security-box">
-                                    <img src={secure2} alt="secure2"/>
-                                    <img src={secure3} alt="secure3"/>
-                                    <img src={secure4} alt="secure4"/>
-                                    <img src={secure5} alt="secure5"/>
-                                    <img src={secure6} alt="secure6"/>
-                                    <img src={secure7} alt="secure7"/>
-                                    <img src={secure8} alt="secure8"/>
-                                    <img src={secure9} alt="secure9"/>
-                                </div>
-                            </div>
+                            <StyledShopww>
```

Comment:
I would just call this StyledShopWorldwide to keep it consistent with the other components